humble/distribution.yaml

```diff
@@ -2161,6 +2161,25 @@ repositories:
       url: https://github.com/ros2/launch_ros.git
       version: humble
     status: maintained
+  ldrobot-lidar-ros2:
+    doc:
+      type: git
+      url: https://github.com/Myzhar/ldrobot-lidar-ros2.git
+      version: humble
+    release:
+      packages:
+      - ldlidar
+      - ldlidar_node
+      - ldlidar_component
+      tags:
+        release: release/humble/{package}/{version}
+      url: https://github.com/ros2-gbp/ldrobot-lidar-ros2.git
+      version: 0.2.0
+    source:
+      type: git
+      url: https://github.com/Myzhar/ldrobot-lidar-ros2.git
+      version: humble
```

Comment:
@Myzhar the `humble` branch doesn't seem to exist in the source (and doc) repository.

Reply:
Adding it

Reply:
Fixed